public/app/features/dashboard/components/TransformationsEditor/TransformationFilter.tsx

```diff
@@ -11,18 +10,20 @@ import {
 import { Field, useStyles2 } from '@grafana/ui';
 import { FrameSelectionEditor } from 'app/plugins/panel/geomap/editor/FrameSelectionEditor';
 
+import { TransformationData } from './TransformationsEditor';
+
 interface TransformationFilterProps {
   index: number;
   config: DataTransformerConfig;
-  data: DataFrame[];
+  data: TransformationData;
   onChange: (index: number, config: DataTransformerConfig) => void;
 }
 
 export const TransformationFilter = ({ index, data, config, onChange }: TransformationFilterProps) => {
   const styles = useStyles2(getStyles);
   const context = useMemo(() => {
     // eslint-disable-next-line
-    return { data } as StandardEditorContext<unknown>;
+    return { data: data.series } as StandardEditorContext<unknown>;
```

Comment:
not sure what TransformationFilter does, but this is probably not sufficient to cover the new functionality?

Reply:
heh @ryantxu i totally forgot this existed. it only shows up in the toolbar when there's more than one dataframe in `series` :rofl: 

i'm gonna stick this source selector in there for now.

![image](https://github.com/grafana/grafana/assets/43234/0ea7b6f3-7699-48d0-babc-f4dafc95e4c7)